server.lua

```diff
@@ -30,12 +30,17 @@ local function GetClosestSafe(Coords)
     return ClosestSafeIndex
 end
 
-local function PoliceAlert(Text, Source)
-    if CalledCops[Source] then return end
-    CalledCops[Source] = true
-    TriggerEvent('police:server:policeAlert', Text)
+local function PoliceAlert(text, source, camId)
```

Comment:
nit; camelCase local function names. Also would be better named as alertPolice. Verb first is usually clearer as to what action the function is doing.
https://manason.github.io/effective-fivem-lua/functions/#naming